bitnami/mongodb/values.yaml

```diff
@@ -1104,3 +1104,483 @@ metrics:
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ##
+    nodePort: ""
+    ## MongoDB(R) service clusterIP IP
+    ##
+    # clusterIP: None
+    ## Specify the externalIP value ClusterIP service type.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#external-ips
+    ##
+    externalIPs: []
+    ## Specify the loadBalancerIP value for LoadBalancer service types.
+    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer
+    ##
+    # loadBalancerIP:
+    ## Specify the loadBalancerSourceRanges value for LoadBalancer service types.
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ##
+    loadBalancerSourceRanges: []
+    ## Provide any additional annotations which may be required. Evaluated as a template
+    ##
+    annotations: {}
+
+  ## External Access to MongoDB(R) nodes configuration
+  ##
+  externalAccess:
+    ## Enable Kubernetes external cluster access to MongoDB(R) nodes
+    ##
+    enabled: false
+    ## External IPs auto-discovery configuration
+    ## An init container is used to auto-detect LB IPs or node ports by querying the K8s API
+    ## Note: RBAC might be required
+    ##
+    autoDiscovery:
+      ## Enable external IP/ports auto-discovery
+      ##
+      enabled: false
+      ## Bitnami Kubectl image
+      ## ref: https://hub.docker.com/r/bitnami/kubectl/tags/
+      ##
+      image:
+        registry: docker.io
+        repository: bitnami/kubectl
+        tag: 1.18.15-debian-10-r24
+        ## Specify a imagePullPolicy
+        ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+        ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+        ##
+        pullPolicy: IfNotPresent
+        ## Optionally specify an array of imagePullSecrets (secrets must be manually created in the namespace)
+        ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+        ## Example:
+        ## pullSecrets:
+        ##   - myRegistryKeySecretName
+        ##
+        pullSecrets: []
+      ## Init Container resource requests and limits
+      ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+      ##
+      resources:
+        # We usually recommend not to specify default resources and to leave this as a conscious
+        # choice for the user. This also increases chances charts run on environments with little
+        # resources, such as Minikube. If you do want to specify resources, uncomment the following
+        # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+        limits: {}
+        #   cpu: 100m
+        #   memory: 128Mi
+        requests: {}
+        #   cpu: 100m
+        #   memory: 128Mi
```

Comment:
I'd use a different approach. Instead of defining the characteristics of the external access for hidden nodes under `hidden.externalAccess.service.*`, I'd use `externalAccess.hiddenService.*`.
This way, you don't need to duplicate the `externalAccess.autoDiscovery` block.